examples/analog_in/cont_voltage_acq_int_clk.py

```diff
@@ -0,0 +1,31 @@
+﻿"""Example of analog input voltage acquisition.
+
+This example demonstrates how to acquire a continuous amount of data
+using the DAQ device's internal clock.
+"""
+
+import pprint
+import time
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+pp = pprint.PrettyPrinter(indent=4, compact=True)
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+    task.timing.cfg_samp_clk_timing(
+        1000.0, sample_mode=AcquisitionType.CONTINUOUS, samps_per_chan=50
```

Comment:
For continuous acquisitions, samps_per_chan is a "hint" of what buffer size to use: https://www.ni.com/docs/en-US/bundle/ni-daqmx/page/mxcncpts/buffersize.html

Specifying 50 has no effect because it is smaller than the default buffer size. With a rate of 1000 S/s, it will use a buffer size of 10 kS.

Reply:
Since we no longer have to print data, `samsps_per_chan` is set to 1000.

Reply:
Noted, `samps_per_chan` is removed from the continuous examples.